source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -233,18 +236,26 @@ export default class CreepVote extends React.Component {
    */
 
   renderSignUp() {
-    return (
+    return [
+      <button
+        className="btn btn-close-sign-up text-uppercase d-flex justify-content-between align-items-center"
+        onClick={() => this.handleSignUp(false)}
+        type="button"
+      >
+        Close
+      </button>,
       <JoinUs
         formPosition="flow"
-        flowHeading={getText(`Thanks for voting! One moment —`)}
+        flowHeading={getText(`You Voted! You Rock!`)}
         flowText={getText(
-          `We strive to protect the internet as a global public resource, but we can only do it with people like you. Join our email list to take action and stay updated!`
+          `Now that you’re on a roll, why not join Mozilla? We’re not creepy (we promise). We actually fight back against creepy. And we need more people like you.`
         )}
+        flowSubText={getText(`(Your results are under here)`)}
```

Comment:
slightly odd phrasing, maybe this instead?
```suggestion
        flowSubText={getText(`(Your results are under this dialog)`)}
```

Reply:
Removed!